pkg/webhook/scheme.go

```diff
@@ -0,0 +1,17 @@
+package webhook
+
+import (
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/jetstack/cert-manager/pkg/apis/certmanager/v1alpha1"
+	"github.com/jetstack/cert-manager/pkg/internal/apis/certmanager/install"
+)
+
+var (
+	Scheme = runtime.NewScheme()
```

Comment:
I've introduced a dedicated scheme for the webhook instead of `pkg/api` - this scheme has the internal API group registered as well as defaulting and conversion functions registered against `v1alpha1` via `github.com/jetstack/cert-manager/pkg/internal/apis/certmanager/install`